.github/actions/openziti-tunnel-build-action/redhat-7/Dockerfile

```diff
@@ -39,7 +42,7 @@ ENV VCPKG_ROOT=/usr/local/vcpkg
 ENV VCPKG_FORCE_SYSTEM_BINARIES=yes
 
 RUN cd /usr/local \
-    && git clone --branch 2023.04.15 https://github.com/microsoft/vcpkg \
+    && git clone --branch 2023.06.20 https://github.com/microsoft/vcpkg \
```

Comment:
we'll need a corresponding PR with this change (and possibly others) in ziti-builder to cover the GH release builds.

Reply:
Can do. Which others can you think of?

Reply:
https://github.com/openziti/ziti-builder/pull/23

Reply:
I'm not sure which other changes will be needed there. I was mainly thinking about comparing the packages that were added here and make sure they are present in ziti-builder?

Once you have a candidate builder image you'll need to reference its version in this PR, yeah?

Reply:
The packages are present in ziti-builder now from a previous PR. We need a newly published image tag that includes those changes and the above PR (the version bump) so we can update .github/workflows/cmake.yml references to point to it.

+cc @qrkourier

Reply:
I just released `ziti-builder:1.0.6`.

Reply:
Build seems to have run successfully. Thanks @qrkourier.